{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -19,6 +19,7 @@ django-bootstrap4 = "~=0.0.8"
 django-allauth = "~=0.39.1"
 django-rest-auth = "~=0.9.5"
 django-extensions = "~=2.1.9"
+django-sendgrid-v5 = "*"
```

Comment:
What does this library add that sending through SMTP doesn't give us?

Reply:
I think maybe not support properly so I was at this package and everything works fine